source/Integration_overview.rst

```diff
@@ -124,46 +124,49 @@ Requests from the client browser require a configured :ref:`CORS policy<cors-pol
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Staging                 | Staging            | An environment that the broker uses during integration development in the TradingView sandbox.                                                      |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Production              | Staging            | An environment that TradingView and the broker use to test any changes the broker made after release to the TradingView production environment.     |
+|                         |                    | Used to ensure that changes do not affect the broker production environment.                                                                        |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Localhost               | Production         | An environment that TradingView developers use for troubleshooting.                                                                                 |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Staging                 | Production         | A pre-release environment that TradingView developers use for testing features implemented by the broker.                                           |
+|                         |                    | During this stage, TradingView performs the final testing of the broker's production API and the deployment of the TradingView client applications. |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Production              | Production         | A live production environment that is ready for users.                                                                                              |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+
+The TradingView website in a staging or production environment can only be connected to one broker environment at a time.
+You can switch between environments through the browser console.
+The tradingView team will provide the steps after they complete the configuration.
```

Comment:
I would explain a bit more what are the `steps`. Didn't come up with a better idea how to rewrite it, but maybe this will help you somehow.
```suggestion
After the TradingView team complete the configuration, you will be informed how to switch between environments through the browser console.
```